web/sites/default/config/block.block.views_block__db_search_group_news_block_featured_blog.yml

```diff
@@ -0,0 +1,42 @@
+uuid: 3432d519-ce0e-42ed-9f07-64a4b3677ebf
+langcode: en
```

Comment:
@ccdatapdx @RichardDavies , this file should not be added back in. I think there may be an issue with the order of git operations.

Reply:
removed this file